unity/Assets/Scripts/Quest/CameraController.cs

```diff
@@ -183,6 +185,8 @@ void FixedUpdate ()
     // Scrolling by mouse/scripts goes here, rate is fixed, display needs to be smooth
     void Update()
     {
+        scrollEnabled = ScrollEnabled();
+
         // latch positions on mouse down
         if (Input.GetMouseButtonDown(0) && ScrollEnabled())
```

Comment:
I overlooked this call to scrollEnable(), this should be replaced with the variable scrollEnabled.